ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/compute/gpu/provision/cucushift-installer-rehearse-nutanix-ipi-compute-gpu-provision-chain.yaml

```diff
@@ -2,6 +2,7 @@ chain:
   as: cucushift-installer-rehearse-nutanix-ipi-compute-gpu-provision
   steps:
   - chain: ipi-conf-nutanix
+  - ref: cucushift-installer-check-nutanix-invalid-fields-gpu
```

Comment:
Why not add the step to `cucushift-installer-check-nutanix-invalid-fields` ?

Reply:
I think this step should run on `cluster_profile: nutanix-qe-gpu`, others run with `cluster_profile: nutanix-qe-zone`, will double check it

Reply:
Confirmed, it can only run on nutanix-qe-gpu